composer.lock

```diff
@@ -645,41 +642,38 @@
         },
         {
             "name": "doctrine/doctrine-bundle",
-            "version": "1.6.13",
+            "version": "1.6.0",
```

Comment:
Is this really last possible version? We have downgraded here.
I can see changes that can prevent us to include last 1.6.x version, but do we really need a 1.6.0?
https://github.com/doctrine/DoctrineBundle/compare/1.6.0...1.6.13

Reply:
That was Composer's decision based on [telling him](https://github.com/mautic/mautic/pull/5718/files#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780R153) Mautic's PHP version requirement. We can try to force some newer version. Just have to figure out which one.